### ISSUE TITLE
[Accessibility] $support-02 value in light themes doesn't meet contrast accessibility

### ISSUE THREAD

**Issue by jeanservaas** [carbon-design-system/carbon#8594]
## [Accessibility] $support-02 value in light themes doesn't meet contrast accessibility

The Accessibility team pointed out that our $support-02 token is not passing contrast accessibility on Gray 10. The value of the token should be updated in the light themes from Green 50 to Green 60 in order to pass and also to be consistent with the Red and Blue support values (both of which are 60).

![image](https://user-images.githubusercontent.com/43144260/117338786-9aae2a00-ae64-11eb-9316-11b591c218d8.png)

### Token update
- [x] Update the value of $support-02 token to be Green 60 (not Green 50) in both the White and Gray 10 themes (hex: #198038)

### Website update

- [x] Update the website: should say Green 60 (not 50) #198038

White 
![image](https://user-images.githubusercontent.com/43144260/117338037-b6fd9700-ae63-11eb-8f29-554db51365ec.png)

G10
![image](https://user-images.githubusercontent.com/43144260/117338086-c67ce000-ae63-11eb-978a-3320095ab566.png)

### Kit update

![image](https://user-images.githubusercontent.com/43144260/117339066-ed87e180-ae64-11eb-97a2-7e5cb7ec3878.png)
